### PR TITLE
Feature 40 update deployment plan add work with migrations

### LINKED ISSUE #40: update buildplan to apply database migrations during deploy

### PR DESCRIPTION
**Pull request by vkhramtsov**
To use this we need to define in admin part of travis-ci next variables:
- `DEPLOY_DB_HOST`
- `DEPLOY_DB_NAME`
- `DEPLOY_DB_USER`
- `DEPLOY_DB_PASSWORD`

Also we need to check deploy process after merging. I think we can do it on weekend.
Closes #40

### CODE REVIEW

File: app/config/parameters_prod.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# This file is auto-generated during the composer install
+parameters:
+    database_host: '~PROJECT_DB_HOST~'
```
Comment (vkhramtsov) [inline app/config/parameters_prod.yml]: Database related configuration will be replaced during build.

File: app/config/parameters_prod.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    database_port: null
+    database_name: '~PROJECT_DB_NAME~'
+    database_user: '~PROJECT_DB_USER~'
+    database_password: '~PROJECT_DB_PASSWORD~'
+    mailer_transport: smtp
+    mailer_host: 127.0.0.1
+    mailer_user: null
+    mailer_password: null
+    secret: '~PROJECT_SECRET~'
```
Comment (vkhramtsov) [inline app/config/parameters_prod.yml]: Secret will be generated while build

File: bin/deploy-to-prod.sh [hunk 3]
```diff
@@ -4,9 +4,7 @@ if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ $(phpenv version-name) = "7.0" ];
     eval "$(ssh-agent -s)"
     chmod 600 .travis/deploy.key
     ssh-add .travis/deploy.key
-    mkdir build
-    mv * build
-    tar -czf package.tgz build
```
Comment (vkhramtsov) [inline bin/deploy-to-prod.sh]: Now phing create package

File: bin/deploy-to-prod.sh [hunk 4]
```diff
@@ -4,9 +4,7 @@ if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ $(phpenv version-name) = "7.0" ];
...
     scp package.tgz $DEPLOY_USER@$DEPLOY_HOST:$DEPLOY_PATH
     ssh $DEPLOY_USER@$DEPLOY_HOST $DEPLOY_PATH/deploy.sh
+    ssh php ~/php.of.by/bin/console doctrine:migrations:migrate
```
Comment (vkhramtsov) [inline bin/deploy-to-prod.sh]: Execute migrations

File: build.xml [hunk 5]
```diff
@@ -3,8 +3,14 @@
 
     <property file="build.properties"/>
     <property name="project-version" value="${env.TRAVIS_TAG}.${env.TRAVIS_BUILD_NUMBER}"/>
+    <property name="project-prod-db-host" value="${env.DEPLOY_DB_HOST}"/>
```
Comment (vkhramtsov) [inline build.xml]: Set up vars for database

File: bin/deploy-to-prod.sh (hunk 4 above)
Comment (chilic) [inline bin/deploy-to-prod.sh]: Looks like missed credential
`ssh php ~/php.of.by/bin/console doctrine:migrations:migrate`

Should be 
`ssh $DEPLOY_USER@$DEPLOY_HOST php ~/php.of.by/bin/console doctrine:migrations:migrate`

Comment (vkhramtsov) [inline bin/deploy-to-prod.sh]: For sure. Fixed

**Pull request merged**